JotunnLib/Managers/LocalizationManager.cs

```diff
@@ -228,32 +228,28 @@ private void AddLanguageFilesFromPluginFolder()
         /// <param name="forceReplace">replace the token if it already exists</param>
         public void AddToken(string token, string value, string language = DefaultLanguage, bool forceReplace = false)
         {
-            if (token[0] != TokenFirstChar)
```

Comment:
Would we be able to get rid of `TokenFirstChar` entirely now? Or is it still being used somewhere

Reply:
Still used in Skillsconfig iirc. Search for "$" (including the quotes)